common/src/main/java/io/confluent/connect/storage/common/StorageCommonConfig.java

```diff
@@ -53,6 +53,8 @@
   public static final String FILE_DELIM_DISPLAY = "File Delimiter";
 
   protected static final ConfigDef CONFIG_DEF = new ConfigDef();
+  public static final GenericClassRecommender STORAGE_CLASS_RECOMMENDER =
+      new GenericClassRecommender();
```

Comment:
The benefit of having the GenericClassRecommender use generics is that this could instead be of type `GenericClassRecommender<Storage<?,?>`, which seems like it'd be safer, if it could work.

Reply:
I agree. But see above restriction re: modules

Reply:
What restriction?

Reply:
Ah, I see the earlier comment. Had to expand the outdated code to see it.

Reply:
Yeah, it was gone because of the `synchronized` keyword.